src/sql/SelectStatement.h

```diff
@@ -96,6 +105,7 @@ struct SelectStatement : SQLStatement {
   std::vector<OrderDescription*>* order;
   std::vector<WithDescription*>* withDescriptions;
   LimitDescription* limit;
+  ForLockingClause* lockings;
```

Comment:
The implementation of this pull request only supports one locking clause. However, multiple can be passed. Thus, we should store multiple locking clauses here.

https://www.postgresql.org/docs/14/sql-select.html

Reply:
Thank you for accepting my patch into the project. I'm working on refining it now. One question here: what do you mean by saying multiple locking clause can be passed here, could you give an SQL query example of this?

Reply:
I think I figured out what you meant. I'm working on it right now.

Reply:
The following query is an example with multiple locking clauses:
```
CREATE TABLE t1 (c1 INT);
CREATE TABLE t2 (c1 INT);
INSERT INTO t1 (c1) VALUES (5);
INSERT INTO t2 (c1) VALUES (3);
SELECT t1.c1 AS t1c1, t2.c1 AS t2c1 FROM t1, t2
FOR UPDATE OF t1 NOWAIT
FOR NO KEY UPDATE OF t2 NOWAIT;
```

Verified with PostgreSQL on https://sqliteonline.com/.